src/all/aboutus/index.njk

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+pagination:
+  data: site.languages
+  size: 1
+  alias: language
+  addAllPagesToCollections: true
+eleventyComputed:
+  title: "{{ i18n[language.locale].aboutUs }}"
+  metaTitle: "{{ site.meta[language.locale].title }} - {{ title }}"
+  locale: "{{ language.locale }}"
+  localeRegion: "{{ language.localeRegion }}"
+  language: "{{ language.localeRegion }}"
+permalink: /{{language.locale}}/aboutus/index.html
+---
+
+{% block content %}
+
+    <section class="mt-15 w-full">
+        <div class="my-8 "></div>
+        <p class="max-w-6xl text-center mx-auto">We are a small group of IT specialists, nerds and enthusiasts who started this blog to share our projects, experiences and passions with you. </p>
```

Comment:
Hey @joshua-schnabel :-)
sollte hier nicht ein Verweis in die AboutUs.json sein? Oder bin ich gerade auf dem Holzweg?